### ISSUE TITLE
how to  Improve accuracy？ it seems the max accuracy is 50% despite I train with 100 epochs

### ISSUE THREAD

**Issue by juzhitao** [TropComplique/ShuffleNet-tensorflow#6]

**Comment by TropComplique** [comment]
Well, it's quite a typical accuracy if you train on Tiny ImageNet from scratch.
If you want bigger accuracy, you need to do a lot of hyperparameter tuning. But I doubt that you can get more than ~55%.

If you want to reproduce the original paper's results, you have to train on
the standard ImageNet dataset.
